src/helpers/hfpQueryManager.js

```diff
@@ -25,11 +23,13 @@ function createHfpItem(rawHfp) {
 
   return {
     ...rawHfp,
+    received_at_unix: moment.tz(rawHfp.received_at, "Europe/Helsinki").unix(),
     journey_start_timestamp: journeyStartMoment.toISOString(),
```

Comment:
@haphut We are adding these props to the fetched HFP items in the UI. Could we at some point get these from the backend? `journey_start_timestamp` is a timestamp like `received_at` but for the journey start time, and `received_at_unix` is a unix timestamp for the `received_at` time.